### ISSUE TITLE
Bug: can't define country, city etc

### ISSUE THREAD

**Issue by aniel300** [qdm12/gluetun#1613]
### Is this urgent?

Yes

### Host OS

ubuntu/ docker

### CPU arch

x86_64

### VPN service provider

PureVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

Running version latest built on 2023-05-25T12:12:44.932Z (commit a43973c)

### What's the problem 🤔

can't define country, city etc as per the instructions here: https://github.com/qdm12/gluetun/wiki/PureVPN

### Share your logs

```plain text
2023-05-26T23:31:17-04:00 INFO Shutdown successful
========================================
========================================
=============== gluetun ================
========================================
=========== Made with ❤️ by ============
======= https://github.com/qdm12 =======
========================================
========================================
Running version latest built on 2023-05-25T12:12:44.932Z (commit a43973c)
🔧 Need help? https://github.com/qdm12/gluetun/discussions/new
🐛 Bug? https://github.com/qdm12/gluetun/issues/new
✨ New feature? https://github.com/qdm12/gluetun/issues/new
☕ Discussion? https://github.com/qdm12/gluetun/discussions/new
💻 Email? quentin.mcgaw@gmail.com
💰 Help me? https://www.paypal.me/qmcgaw https://github.com/sponsors/qdm12
2023-05-26T23:32:17-04:00 INFO [routing] default route found: interface eth0, gateway 172.18.0.1, assigned IP 172.18.0.13 and family v4
2023-05-26T23:32:17-04:00 INFO [routing] local ethernet link found: eth0
2023-05-26T23:32:17-04:00 INFO [routing] local ipnet found: 172.18.0.0/16
2023-05-26T23:32:17-04:00 INFO [firewall] enabling...
2023-05-26T23:32:17-04:00 INFO [firewall] enabled successfully
2023-05-26T23:32:18-04:00 INFO [storage] merging by most recent 13056 hardcoded servers and 13056 servers read from /gluetun/servers.json
2023-05-26T23:32:18-04:00 ERROR VPN settings: provider settings: server selection: for VPN service provider purevpn: the country specified is not valid: value is not one of the possible choices: value germany, choices available are Albania, Australia, Austria, Belgium, Brazil, Bulgaria, Canada, Costa Rica, Czech Republic, Denmark, Estonia, Finland, France, Georgia, Germany, Greece, Hong Kong, Hungary, Iceland, Indonesia, Ireland, Israel, Italy, Japan, Kenya, Korea, Latvia, Lithuania, Luxembourg, Malaysia, Mexico, Moldova, Netherlands, New Zealand, Norway, Poland, Portugal, Romania, Russian Federation, Serbia, Singapore, Slovakia, South Africa, Spain, Sweden, Switzerland, Taiwan, Thailand, Turkey, Ukraine, United Arab Emirates, United Kingdom, United States
2023-05-26T23:32:18-04:00 INFO Shutdown successful
```

### Share your configuration

```yml
gluetun_vpn:
    restart: unless-stopped
    container_name: gluetun_vpn
    ports:
      - 8888:8888 # HTTP proxy
      - 8388:8388 # Shadowsocks
    # dns:
    #   - 1.1.1.1
    image: qmcgaw/gluetun
    devices:
      - /dev/net/tun:/dev/net/tun    
    cap_add:
      - NET_ADMIN
    volumes:
      - $PERSIST/gluetun_vpn:/gluetun
    environment:
      # See https://github.com/qdm12/gluetun/wiki
      - VPN_SERVICE_PROVIDER=purevpn
      - VPN_TYPE=openvpn
      - OPENVPN_USER=
      - OPENVPN_PASSWORD=
      - SERVER_COUNTRIES=Germany
      #- SERVER_REGIONS=Hesse
      #- SERVER_HOSTNAMES=de2-auto-tcp.ptoserver.com,de2-auto-udp-obf.ptoserver.com
      # Wireguard:
      # - WIREGUARD_PRIVATE_KEY=wOEI9rqqbDwnN8/Bpp22sVz48T71vJ4fYmFWujulwUU=
      # - WIREGUARD_ADDRESS=10.64.222.21/32
      # Timezone for accurate log times
      - PUID=$PUID
      - PGID=$PGID
      - TZ=$TIME_ZONE
      - UPDATER_PERIOD=24h
```

**Comment by aniel300** [comment]
can confirm that v3.34.0 but not latest version

**Comment by rigforsilentrunning** [comment]
can confirm on version "2023-05-25T12:12:44.932Z (commit a43973c)" It appears to be sanitizing the input region to all lowercase.

```yaml
image: qmcgaw/gluetun:latest
    cap_add:
      - NET_ADMIN
    environment:
      - SERVER_REGIONS=Switzerland
      #- REGION=Switzerland
      - OPENVPN_USER=
      - OPENVPN_PASSWORD=
      - VPN_SERVICE_PROVIDER=private internet access
```


```logs
2023-05-27T05:05:41Z ERROR VPN settings: provider settings: server selection: for VPN service provider private internet access: the region specified is not valid: value is not one of the possible choices: value switzerland, choices available are AU Australia Streaming Optimized, AU Brisbane, AU Melbourne, AU Perth, AU Sydney, Albania, Algeria, Andorra, Argentina, Armenia, Austria, Bahamas, Bangladesh, Belgium, Bosnia and Herzegovina, Brazil, Bulgaria, CA Montreal, CA Ontario, CA Toronto, CA Vancouver, Cambodia, Chile, China, Colombia, Costa Rica, Croatia, Cyprus, Czech Republic, DE Berlin, DE Frankfurt, DK Copenhagen, DK Streaming Optimized, ES Madrid, ES Valencia, Egypt, Estonia, FI Helsinki, FI Streaming Optimized, France, Georgia, Greece, Greenland, Hong Kong, Hungary, IT Milano, IT Streaming Optimized, Iceland, India, Indonesia, Ireland, Isle of Man, Israel, JP Streaming Optimized, JP Tokyo, Kazakhstan, Latvia, Liechtenstein, Lithuania, Luxembourg, Macao, Malaysia, Malta, Mexico, Moldova, Monaco, Mongolia, Montenegro, Morocco, Netherlands, New Zealand, Nigeria, North Macedonia, Norway, Panama, Philippines, Poland, Portugal, Qatar, Romania, SE Stockholm, SE Streaming Optimized, Saudi Arabia, Serbia, Singapore, Slovakia, Slovenia, South Africa, Sri Lanka, Switzerland, Taiwan, Turkey, UK London, UK Manchester, UK Southampton, UK Streaming Optimized, US Alabama, US Alaska, US Arkansas, US Atlanta, US Baltimore, US California, US Chicago, US Connecticut, US Denver, US East, US East Streaming Optimized, US Florida, US Honolulu, US Houston, US Idaho, US Indiana, US Iowa, US Kansas, US Kentucky, US Las Vegas, US Louisiana, US Maine, US Massachusetts, US Michigan, US Minnesota, US Mississippi, US Missouri, US Montana, US Nebraska, US New Hampshire, US New Mexico, US New York, US North Carolina, US North Dakota, US Ohio, US Oklahoma, US Oregon, US Pennsylvania, US Rhode Island, US Salt Lake City, US Seattle, US Silicon Valley, US South Carolina, US South Dakota, US Tennessee, US Texas, US Vermont, US Virginia, US Washington DC, US West, US West Streaming Optimized, US West Virginia, US Wilmington, US Wisconsin, US Wyoming, Ukraine, United Arab Emirates, Venezuela, Vietnam
```

**Comment by aniel300** [comment]
> can confirm on version "2023-05-25T12:12:44.932Z (commit [a43973c](https://github.com/qdm12/gluetun/commit/a43973c093662c2b5521eb2e9e3aea053584e96b))" It appears to be sanitizing the input region to all lowercase.
> 
> ```yaml
> image: qmcgaw/gluetun:latest
>     cap_add:
>       - NET_ADMIN
>     environment:
>       - SERVER_REGIONS=Switzerland
>       #- REGION=Switzerland
>       - OPENVPN_USER=
>       - OPENVPN_PASSWORD=
>       - VPN_SERVICE_PROVIDER=private internet access
> ```
> 
> ```
> 2023-05-27T05:05:41Z ERROR VPN settings: provider settings: server selection: for VPN service provider private internet access: the region specified is not valid: value is not one of the possible choices: value switzerland, choices available are AU Australia Streaming Optimized, AU Brisbane, AU Melbourne, AU Perth, AU Sydney, Albania, Algeria, Andorra, Argentina, Armenia, Austria, Bahamas, Bangladesh, Belgium, Bosnia and Herzegovina, Brazil, Bulgaria, CA Montreal, CA Ontario, CA Toronto, CA Vancouver, Cambodia, Chile, China, Colombia, Costa Rica, Croatia, Cyprus, Czech Republic, DE Berlin, DE Frankfurt, DK Copenhagen, DK Streaming Optimized, ES Madrid, ES Valencia, Egypt, Estonia, FI Helsinki, FI Streaming Optimized, France, Georgia, Greece, Greenland, Hong Kong, Hungary, IT Milano, IT Streaming Optimized, Iceland, India, Indonesia, Ireland, Isle of Man, Israel, JP Streaming Optimized, JP Tokyo, Kazakhstan, Latvia, Liechtenstein, Lithuania, Luxembourg, Macao, Malaysia, Malta, Mexico, Moldova, Monaco, Mongolia, Montenegro, Morocco, Netherlands, New Zealand, Nigeria, North Macedonia, Norway, Panama, Philippines, Poland, Portugal, Qatar, Romania, SE Stockholm, SE Streaming Optimized, Saudi Arabia, Serbia, Singapore, Slovakia, Slovenia, South Africa, Sri Lanka, Switzerland, Taiwan, Turkey, UK London, UK Manchester, UK Southampton, UK Streaming Optimized, US Alabama, US Alaska, US Arkansas, US Atlanta, US Baltimore, US California, US Chicago, US Connecticut, US Denver, US East, US East Streaming Optimized, US Florida, US Honolulu, US Houston, US Idaho, US Indiana, US Iowa, US Kansas, US Kentucky, US Las Vegas, US Louisiana, US Maine, US Massachusetts, US Michigan, US Minnesota, US Mississippi, US Missouri, US Montana, US Nebraska, US New Hampshire, US New Mexico, US New York, US North Carolina, US North Dakota, US Ohio, US Oklahoma, US Oregon, US Pennsylvania, US Rhode Island, US Salt Lake City, US Seattle, US Silicon Valley, US South Carolina, US South Dakota, US Tennessee, US Texas, US Vermont, US Virginia, US Washington DC, US West, US West Streaming Optimized, US West Virginia, US Wilmington, US Wisconsin, US Wyoming, Ukraine, United Arab Emirates, Venezuela, Vietnam
> ```

v3.34.0 works fine fyi

**Comment by qdm12** [comment]
The latest image `qmcgaw/gluetun` was fixed a few minutes ago by 7e325715c7a24da73447b29cec94b207bd950745.
The regression was introduced 2 days ago with a43973c093662c2b5521eb2e9e3aea053584e96b which did compare text taking care about case sensitivity (gluetun lowercases everything inputted, but keeps servers data text with case sensitivity). My apologies for the delay fixing it, obviously, and thank you for reporting the issue 🎖️ 

Just in case you are not aware, you can use released docker images such as `qmcgaw/gluetun:v3` or `qmcgaw/gluetun:v3.34.1` which should be more stable than the latest image `qmcgaw/gluetun`. I always appreciate users using the latest image since this helps fix bugs before bringing them to a stable release which would end up unstable, so ideally keep using the latest image 😉

**Comment by aniel300** [comment]
thank you
